docs/cloud/how-to/collaborate-using-orgs.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Collaborate using Organizations"
+description: "A short guide on colaboration using organizations"
+sidebar_position: 4
+---
+
+Organizations allow teams to collaborate by sharing access to clusters and private hub packages.
+
+The user that creates an organziaton becomes its owner and all invited users are considered members. Cluster creation/deletion and user management is limited to the owner. 
+Members have permission to perform all other operations.
+
+At the moment access control within an organization is not configurable. 
+Therefore, in cases where you need to segment access to resources, we recommend that you create an organization per segment.
+
+Members of an organization may publish InfinyOn Hub packages to be made availabile to the organization.
+Any private packages published to this organization will be available to all its cluster and users.
```

Comment:
| Any private packages published to this organization will be available to all its cluster and users.
Any private packages published to this organization will be available to all its clusters and users.

Reply:
Fixed, made "cluster" -> "clusters" plural. Is that is?